src/app/models/Students.js

```diff
@@ -1,6 +1,6 @@
 import Sequelize, { Model } from 'sequelize';
 
-class Student extends Model {
+class Students extends Model {
```

Comment:
I think that is better use singular word for model name

Reply:
It doesn't matter, depends the use too.

What matters, however, is that you pick a scheme and be consistent.